llvm/test/MC/WebAssembly/eh-assembly.s

```diff
@@ -24,7 +24,6 @@ eh_test:
         return
       end_block
       throw_ref
-      drop
```

Comment:
After `throw_ref`, the stack is polymorphic, so you don't need to drop the `i32`